src/emulator/host/src/config.cpp

```diff
@@ -100,6 +100,22 @@ ExitCode init(Config &cfg, int argc, char **argv) {
 
         logging::set_level(static_cast<spdlog::level::level_enum>(*cfg.log_level));
 
+        LOG_INFO_IF(cfg.vpk_path, "input-vpk-path: {}", *cfg.vpk_path);
+        LOG_INFO_IF(cfg.run_title_id, "input-installed-id: {}", *cfg.run_title_id);
+        if (!cfg.lle_modules.empty()) {
+            auto modules = std::string();
```

Comment:
Bit of an unorthodox way to declare/init, just do this which has the same effect:
```cpp
std::string modules;
```

Reply:
Thanks!